src/blocks/media-block/ImageBlock.test.tsx

```
@@ -0,0 +1,211 @@
+  onload: (() => void) | null = null;
+  onerror: (() => void) | null = null;
+  src = '';
+  width = 800;
+  height = 600;
+  naturalWidth = 800;
+  naturalHeight = 600;
+  complete = false;
+  
+  constructor() {
+    // Store a reference to 'this' to avoid TypeScript issues
+    const self = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)